generators/server/templates/src/main/java/package/config/JacksonConfiguration.java.ejs

```diff
@@ -71,4 +72,13 @@ public class JacksonConfiguration {
     public ConstraintViolationProblemModule constraintViolationProblemModule() {
         return new ConstraintViolationProblemModule();
     }
+
+    /**
+     * Force create default Spring Boot jackson mapper to load custom modules, broken since Spring Boot 2.5.4
+     */
+    @Bean
+    @Primary
+    ObjectMapper jacksonObjectMapper(Jackson2ObjectMapperBuilder builder) {
+        return builder.createXmlMapper(false).build();
+    }
```

Comment:
I couldn't see anything in the changelog and in the source code that indicates this is required.
Is this a couchbase only config?

Reply:
I don't know, I didn't have time to test other configs. But I suppose that there is some new configuration that overrides SB default Object Mapper. There is a lot of issues that talking about this, for example: https://github.com/spring-projects/spring-boot/issues/26859

Reply:
Can we drop to try to find the cause?
I think this breaks the backend to match a broken test.
Probably this https://github.com/jhipster/generator-jhipster/blob/3a2e16cab39dad196cbaebf52fe4d48d0f6f9c77/generators/server/templates/src/test/java/package/web/rest/TestUtil.java.ejs#L67 should use a injected ObjectMapper instead of a created one.
```suggestion
```

Reply:
No, it’s not just tests, it fails also on run. You can test locally without
it if you have time. I’ll try to dig when I have time. I prefer to work
first on reactive part.

Reply:
Can you move this change to CouchbaseConfig so we can merge it?
Jackson doesn’t looks broken at others databases.

Reply:
I found the bug, I created an issue about it: https://github.com/spring-projects/spring-data-couchbase/issues/1209.

I didn't find a method to disable this bean, so the only way is to keep our `jacksonObjectMapper`. Do you want me to move it to `DatabaseConfiguration` for now ?

Reply:
Great find.
Yes please, we should only add this where it is required.